scripts/backup-tor-keys.sh

```
@@ -0,0 +1,294 @@
+    # Create tar archive
+    tar -czf "$ARCHIVE_NAME" "backup-${TIMESTAMP}/"
+
+    # Prompt for passphrase
+    log_info "Enter passphrase for encryption:"
+    gpg --symmetric --cipher-algo AES256 "$ARCHIVE_NAME"
+
+    # Remove unencrypted archive
+    rm -f "$ARCHIVE_NAME"
+
+    # Calculate checksum of encrypted file
+    local CHECKSUM=$(sha256sum "$ENCRYPTED_ARCHIVE" | cut -d' ' -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)